data/NETWORK_LOOP_FLOW_WITH_COUNTRIES.uct

```diff
@@ -0,0 +1,19 @@
+##C 2007.05.01
+This is a test network with outside core countries.
+It is used for loop flow decomposition.
+##N
+##ZES
+EGEN  11 GEN          0 3 400.00 0.00000 0.00000 -100.00 0.00000 1000.00 -1000.0 1000.00 -1000.0
+ELOAD 11 LOAD         0 0        100.000 0.00000 0.00000 0.00000
+##ZFR
+FGEN  11 GEN          0 2 400.00 0.00000 0.00000 -100.00 0.00000 1000.00 -1000.0 1000.00 -1000.0
+FLOAD 11 LOAD         0 0        100.000 0.00000 0.00000 0.00000
+##ZBE
+BGEN  11 GEN          0 2 400.00 0.00000 0.00000 -100.00 0.00000 1000.00 -1000.0 1000.00 -1000.0
+BLOAD 11 LOAD         0 0        100.000 0.00000 0.00000 0.00000
+##L
+EGEN  11 FGEN  11 1 0 1.0000 0.0500 0.000000    480 LINE
+FGEN  11 BGEN  11 1 0 1.0000 0.0500 0.000000    480 LINE
+BGEN  11 BLOAD 11 1 0 1.0000 0.0500 0.000000    480 LINE
+BLOAD 11 FLOAD 11 1 0 1.0000 0.0500 0.000000    480 LINE
+FLOAD 11 ELOAD 11 1 0 1.0000 0.0500 0.000000    480 LINE
```

Comment:
git seems to need a new line at the end of the file

Reply:
done